diff --git a/node_modules/@astrojs/starlight/integrations/code-rtl-support.ts b/node_modules/@astrojs/starlight/integrations/code-rtl-support.ts
index 781b7af..4d5912a 100644
--- a/node_modules/@astrojs/starlight/integrations/code-rtl-support.ts
+++ b/node_modules/@astrojs/starlight/integrations/code-rtl-support.ts
@@ -21,7 +21,7 @@ export function rehypeRtlCodeSupport() {
 			if (el.tagName === 'pre' || el.tagName === 'code') {
 				el.properties ||= {};
 				if (!('dir' in el.properties)) {
-					el.properties.dir = { pre: 'ltr', code: 'auto' }[el.tagName];
+					el.properties["dir"] = { pre: 'ltr', code: 'auto' }[el.tagName];
 				}
 				return SKIP;
 			}
diff --git a/node_modules/@astrojs/starlight/integrations/heading-links.ts b/node_modules/@astrojs/starlight/integrations/heading-links.ts
index ebb61d4..0c9d5bd 100644
--- a/node_modules/@astrojs/starlight/integrations/heading-links.ts
+++ b/node_modules/@astrojs/starlight/integrations/heading-links.ts
@@ -37,7 +37,7 @@ export default function rehypeAutolinkHeadings(
 		const t = useTranslationsForLang(pageLang);
 
 		visit(tree, 'element', function (node, index, parent) {
-			if (!headingRank(node) || !node.properties.id || typeof index !== 'number' || !parent) {
+			if (!headingRank(node) || !node.properties["id"] || typeof index !== 'number' || !parent) {
 				return;
 			}
 
@@ -56,7 +56,7 @@ export default function rehypeAutolinkHeadings(
 				{
 					type: 'element',
 					tagName: 'a',
-					properties: { class: 'sl-anchor-link', href: '#' + node.properties.id },
+					properties: { class: 'sl-anchor-link', href: '#' + node.properties["id"] },
 					children: [AnchorLinkIcon, h('span', { class: 'sr-only' }, accessibleLabel)],
 				}
 			);
diff --git a/node_modules/@astrojs/starlight/user-components/rehype-file-tree.ts b/node_modules/@astrojs/starlight/user-components/rehype-file-tree.ts
index 72f658a..9f14dd4 100644
--- a/node_modules/@astrojs/starlight/user-components/rehype-file-tree.ts
+++ b/node_modules/@astrojs/starlight/user-components/rehype-file-tree.ts
@@ -95,8 +95,8 @@ const fileTreeProcessor = rehype()
 				}
 
 				// Add classes and data attributes to the list item node.
-				node.properties.class = isDirectory ? 'directory' : 'file';
-				if (isPlaceholder) node.properties.class += ' empty';
+				node.properties["class"] = isDirectory ? 'directory' : 'file';
+				if (isPlaceholder) node.properties["class"] += ' empty';
 
 				// Create the tree entry node that contains the icon, file name and comment which will end up
 				// as the list item’s children.
diff --git a/node_modules/@astrojs/starlight/utils/git.ts b/node_modules/@astrojs/starlight/utils/git.ts
index f05fd88..67998f2 100644
--- a/node_modules/@astrojs/starlight/utils/git.ts
+++ b/node_modules/@astrojs/starlight/utils/git.ts
@@ -29,11 +29,11 @@ export function getNewestCommitDate(file: string): Date {
 	const regex = /^(?<timestamp>\d+)$/;
 	const match = output.match(regex);
 
-	if (!match?.groups?.timestamp) {
+	if (!match?.groups?.["timestamp"]) {
 		throw new Error(`Failed to validate the timestamp for file "${file}"`);
 	}
 
-	const timestamp = Number(match.groups.timestamp);
+	const timestamp = Number(match.groups["timestamp"]);
 	const date = new Date(timestamp * 1000);
 	return date;
 }
