diff --git a/node_modules/starlight-theme-obsidian/index.ts b/node_modules/starlight-theme-obsidian/index.ts
index f8090c2..1660f11 100644
--- a/node_modules/starlight-theme-obsidian/index.ts
+++ b/node_modules/starlight-theme-obsidian/index.ts
@@ -5,7 +5,7 @@ import { type StarlightThemeObsidianConfig, validateConfig } from 'starlight-the
 export default function plugin(userConfig?: StarlightThemeObsidianConfig): StarlightPlugin {
 	const parsedConfig = validateConfig(userConfig);
 
-	let i18nArgs = null;
+	let i18nArgs: Parameters<NonNullable<StarlightPlugin['hooks']['i18n:setup']>>[0] | null = null;
 	return {
 		name: 'starlight-theme-obsidian',
 		hooks: {
@@ -23,7 +23,7 @@ export default function plugin(userConfig?: StarlightThemeObsidianConfig): Starl
 				} else {
 					// TODO: This is dumb, bad, and I should really start work on that `addPlugin` PR...
 					const starlightPlugin = starlightSiteGraph(parsedConfig);
-					await starlightPlugin.hooks['i18n:setup']?.(i18nArgs);
+					await starlightPlugin.hooks['i18n:setup']?.(i18nArgs!);
 					await starlightPlugin.hooks['config:setup']?.(args);
 				}
 				const customCss: typeof config.customCss = [
