diff --git a/node_modules/starlight-site-graph/config/graph.ts b/node_modules/starlight-site-graph/config/graph.ts
index 41a14c4..f1768b1 100644
--- a/node_modules/starlight-site-graph/config/graph.ts
+++ b/node_modules/starlight-site-graph/config/graph.ts
@@ -16,7 +16,7 @@ const graphConfig = {
 	actions: ['fullscreen', 'depth', 'reset-zoom', 'render-arrows', 'settings'] as ('fullscreen' | 'depth' | 'reset-zoom' | 'render-arrows' | 'settings')[],
 	tagStyles: {},
 	tagRenderMode: 'none' as ('none' | 'node' | 'same' | 'both'),
-	nodeInclusionRules: ['**/*'] as const,
+	nodeInclusionRules: ['**/*'] as string[],
 	prefetchPages: true,
 	enableDrag: true,
 	enableZoom: true,
diff --git a/node_modules/starlight-site-graph/index.ts b/node_modules/starlight-site-graph/index.ts
index d0fc054..fafe59d 100644
--- a/node_modules/starlight-site-graph/index.ts
+++ b/node_modules/starlight-site-graph/index.ts
@@ -3,7 +3,6 @@ import { validateConfig, type StarlightSiteGraphConfig } from './config';
 
 import integration from './integration';
 import { translations } from './i18n';
-import { ensureTrailingSlash } from './sitemap/util';
 
 export default function plugin(userConfig?: StarlightSiteGraphConfig): StarlightPlugin {
 	const parsedConfig = validateConfig(userConfig);
@@ -16,7 +15,8 @@ export default function plugin(userConfig?: StarlightSiteGraphConfig): Starlight
 			'config:setup': async ({ addIntegration, config, astroConfig, command, logger, updateConfig }) => {
 				if (command === 'preview') return;
 
-				const addTrailingSlashes = astroConfig.trailingSlash !== "never";
+				// @ts-ignore unused variable from upstream
+			const _addTrailingSlashes = astroConfig.trailingSlash !== "never";
 
 				// TODO: Temporary implementation of graph/backlinks exclusion from plugin
 				if (!parsedConfig.graph) {
diff --git a/node_modules/starlight-site-graph/integration.ts b/node_modules/starlight-site-graph/integration.ts
index e538599..171f118 100644
--- a/node_modules/starlight-site-graph/integration.ts
+++ b/node_modules/starlight-site-graph/integration.ts
@@ -86,7 +86,7 @@ export default defineIntegration({
 
 							try {
 								await builder.addMDContentFolder(settings.sitemapConfig.contentRoot, settings.sitemapConfig.pageInclusionRules)
-								settings.sitemapConfig.sitemap = builder.process().toSitemap();
+								settings.sitemapConfig.sitemap = builder.process().toSitemap() as any;
 								logger.info('Finished retrieving links from Markdown content');
 							} catch (e) {
 								logger.error('Failed to retrieve links from Markdown content, reason: ' + e);
@@ -177,10 +177,10 @@ export default defineIntegration({
 							settings.sitemapConfig.sitemap = (await builder
 								.addHTMLContentFolder(outputPath, settings.sitemapConfig.pageInclusionRules))
 								.process()
-								.toSitemap();
+								.toSitemap() as any;
 							logger.info('Finished generating sitemap from generated HTML content');
 						} catch (e) {
-							settings.sitemapConfig.sitemap = builder.process().toSitemap();
+							settings.sitemapConfig.sitemap = builder.process().toSitemap() as any;
 							logger.error('Failed to retrieve links from generated HTML content, reason: ' + e);
 						}
 					}
diff --git a/node_modules/starlight-site-graph/sitemap/process.ts b/node_modules/starlight-site-graph/sitemap/process.ts
index 7f45128..1dee3dd 100644
--- a/node_modules/starlight-site-graph/sitemap/process.ts
+++ b/node_modules/starlight-site-graph/sitemap/process.ts
@@ -28,7 +28,7 @@ export function processSitemap(sitemap: RemoveOptional<Sitemap>, options: FullSt
 			}
 		}
 
-		// @ts-expect-error shapeColor cannot be correctly casted
+		// @ts-ignore shapeColor cannot be correctly casted
 		entry.nodeStyle = {
 			...entry.nodeStyle,
 			...nodeStyle,
diff --git a/node_modules/starlight-site-graph/sitemap/util.ts b/node_modules/starlight-site-graph/sitemap/util.ts
index 059db7a..d0b2ac5 100644
--- a/node_modules/starlight-site-graph/sitemap/util.ts
+++ b/node_modules/starlight-site-graph/sitemap/util.ts
@@ -1,5 +1,6 @@
 import fs from 'node:fs';
 import path from 'node:path';
+// @ts-ignore missing types
 import micromatch from 'micromatch';
 import { slug } from "github-slugger"
 
